### ISSUE TITLE
Finetuning PlaneAE on a new dataset degrades segmentation performance compared to pretrained model

### ISSUE THREAD

**Issue by Okay0805** [svip-lab/PlanarReconstruction#51]
Hi, thanks for releasing PlaneAE, it's a very interesting work.

I am finetuning the official PlaneAE model on a new indoor plane segmentation dataset
(using the official code and pretrained model provided in this repo).

However, I observe a consistent degradation in plane segmentation performance after finetuning,
compared to directly evaluating the pretrained model on the same dataset.

My setup:
Official PlaneAE code (PlanarReconstruction-master)
Pretrained model provided by the authors
Finetuning strategy:
  Backbone / encoder frozen
  Decoder + heads unfrozen
  BatchNorm frozen
  Adam optimizer, lr = 1e-5
  Finetuning epochs: 15 / 40
  Losses: original losses with balanced re-weighting (embedding, mask, depth, param)

Results (example):
Pretrained model (no finetuning): significantly better RI / VI / SC
Finetuned model: RI drops from ~0.8+ to ~0.67, VI increases, SC decreases

It seems that finetuning may destroy the embedding structure learned during pretraining,
especially for large planar regions.

Qualitative comparison of plane segmentation results on the same input image using the official visualization code, showing (from top to bottom): the pretrained PlaneAE model, finetuning with frozen backbone (25 epochs), partial backbone unfreezing (40 epochs), and finetuning with re-weighted losses (15 epochs).

<img width="413" height="408" alt="Image" src="https://github.com/user-attachments/assets/b945f751-f413-493d-ab08-747bdcfd0f1e" />

<img width="407" height="409" alt="Image" src="https://github.com/user-attachments/assets/9e115a95-afda-4f02-8654-0dba9ce9e3ae" />

<img width="407" height="408" alt="Image" src="https://github.com/user-attachments/assets/b11c5fbf-98b0-4d0a-921d-71a6f5b5ebcd" />

<img width="409" height="408" alt="Image" src="https://github.com/user-attachments/assets/27dfb4ac-3bea-497c-a48a-d462df306ff4" />

Questions:
1. Is PlaneAE intended to be finetuned on new datasets?
2. Are there recommended strategies for finetuning the embedding loss
   (e.g., freezing embedding head, different pull/push margins)?
3. Have you observed similar behavior where finetuning degrades segmentation quality?

Any suggestions would be greatly appreciated.
Thanks again for your work!
